grails-app/controllers/org/pih/warehouse/product/ProductAssociationController.groovy

```diff
@@ -73,6 +73,15 @@ class ProductAssociationController {
     def save = {
         def productAssociationInstance = new ProductAssociation(params)
         if (productAssociationInstance.save(flush: true)) {
+            if (params.hasMutualAssociation) {
+                def mutualAssociationInstance = new ProductAssociation()
+                bindMutualAssociationData(mutualAssociationInstance, params)
+
+                mutualAssociationInstance.mutualAssociation = productAssociationInstance
+                productAssociationInstance.mutualAssociation = mutualAssociationInstance
+
+                mutualAssociationInstance.save(flush: true, failOnError: true)
```

Comment:
Again we should add something to undo these changes if the user unchecks the checkbox. Ask Kelsey what she thinks.